### PR TITLE
Farm migrations

### PR DESCRIPTION
**Pull request by Chef-Chungus**

### CODE REVIEW

File: src/config/constants/farms.ts [hunk 1]
```diff
@@ -12,6 +12,1129 @@ const farms: FarmConfig[] = [
...
+    lpAddresses: {
+      97: '',
+      56: '0x9bdEdb0c876fC0Da79D945DF28942b898Af89Fc7',
+    },
+    token: tokens.busd,
+    quoteToken: tokens.wbnb,
+  },
+  /**
+   * All farms below here are from v1 and are to be set to 0x
+   */
```
Comment (Chef-Chungus) [inline src/config/constants/farms.ts:1137]: I added commenting here - hope its ok

File: src/config/constants/farms.ts [hunk 2]
```diff
@@ -12,6 +12,1129 @@ const farms: FarmConfig[] = [
...
+    lpSymbol: 'τBTC-BTCB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xFD09CDbd6A7dCAd8AC47df4F139443a729264763',
+    },
+    token: tokens.τbtc,
+    quoteToken: tokens.btcb,
+  },
+  {
+    pid: 193,
```
Comment (Chef-Chungus) [inline src/config/constants/farms.ts:36]: this is out of order because we recently promoted it

File: src/config/constants/farms.ts [hunk 3]
```diff
@@ -12,6 +12,1129 @@ const farms: FarmConfig[] = [
...
+    pid: 195,
+    lpSymbol: 'SUSHI-ETH LP',
+    lpAddresses: {
+      97: '',
+      56: '0x3BECbb09F622187B544C0892EeDeB58C004117e1',
+    },
+    token: tokens.sushi,
+    quoteToken: tokens.eth,
+  },
+  {
```
Comment (ChefHutch) [inline src/config/constants/farms.ts:595]: Anyone checking - 193 missing because it's the SWINGBY farm, highlighted near the top of the file

**Review by ChefHutch — APPROVED** [review]
Can't validate the addresses but the tokens, LP symbols & pids appear in order 👍

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**